contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -591,6 +591,33 @@ saved. For more details on how to use Merge Patch, see the RFC.
 detailed explanation of how it works and why it needed to be introduced, see
 [here](/contributors/devel/sig-api-machinery/strategic-merge-patch.md).
 
+#### Declaring Patch Strategy
+In the context of [strategic merge patch](/contributors/devel/sig-api-machinery/strategic-merge-patch.md), a set of go markers can be used by API authors to declare the patch strategy of collections of fields (lists, maps and structs).
```

Comment:
Some of these directives are only relevant for server-side apply, not strategic merge patch. @apelisse, can you give guidance on which apply to which?

Reply:
Agreed, I made pretty much the same statement above.